src/rpc/client.zig

```diff
@@ -282,6 +301,9 @@ pub const Client = struct {
             break;
         }
 
+        // sometimes the response is "x-y" where our fields follow "x_y" format, so this
+        // replaces "-" with "_" so we can properly parse the response
+        _ = std.mem.replace(u8, response.bytes.items, "-", "_", response.bytes.items);
```

Comment:
This seems potentially risky in a general request method, is there an issue with a field name in the RPC data structure which uses hyphens? I thought they were camel case. Anyhow I think it would make more sense to use the same field name as the Rpc response using the @"<name>" in this case. Let me know what you think